src/utils/masks.js

```
@@ -0,0 +1,24 @@
+import createNumberMask from 'text-mask-addons/dist/createNumberMask';
+
+export const MASKS = {
+  phone: [
```

Do we want a default phone mask that includes country code?